controllers/api/auth.go

```diff
@@ -0,0 +1,47 @@
+package api_controllers
+
+import (
+	"go-google-scraper-challenge/forms"
+	oauth_services "go-google-scraper-challenge/services/oauth"
+
+	"gopkg.in/oauth2.v3/errors"
+)
+
+// AuthController operations for User
+type AuthController struct {
+	BaseController
+}
+
+// URLMapping map user controller actions to functions
+func (c *AuthController) URLMapping() {
+	c.Mapping("Login", c.Login)
+}
+
+var (
```

Comment:
Why not constants?

Reply:
@olivierobert Because the `errors.Descriptions` is not a constant and it going to fail the lint.
```
// Descriptions error description
var Descriptions = map[error]string{
	ErrInvalidRequest:          "The request is missing a required parameter, includes an invalid parameter value, includes a parameter more than once, or is otherwise malformed",
	ErrUnauthorizedClient:      "The client is not authorized to request an authorization code using this method",
	ErrAccessDenied:            "The resource owner or authorization server denied the request",
	ErrUnsupportedResponseType: "The authorization server does not support obtaining an authorization code using this method",
	ErrInvalidScope:            "The requested scope is invalid, unknown, or malformed",
	ErrServerError:             "The authorization server encountered an unexpected condition that prevented it from fulfilling the request",
	ErrTemporarilyUnavailable:  "The authorization server is currently unable to handle the request due to a temporary overloading or maintenance of the server",
	ErrInvalidClient:           "Client authentication failed",
	ErrInvalidGrant:            "The provided authorization grant (e.g., authorization code, resource owner credentials) or refresh token is invalid, expired, revoked, does not match the redirection URI used in the authorization request, or was issued to another client",
	ErrUnsupportedGrantType:    "The authorization grant type is not supported by the authorization server",
}
```